vendor/double-conversion/upstream/test/cctest/test-conversions.cc

```diff
@@ -1735,6 +1735,33 @@ static double StrToD16(const uc16* str16, int length, int flags,
 }
 
 
+static double StrToD16(const char* str, int flags,
+                       double empty_string_value,
+                       int* processed_characters_count, bool* processed_all,
+                       char char_separator, uc16 separator) {
+  uc16 str16[256];
+  int length = -1;
+  for (int i = 0;; i++) {
```

Comment:
If the input string `str` is longer than 256 bytes, it will cause a stack buffer over-*write*.
Fortunately, this is just test code though...

Reply:
True, but we don't actually run this code.  It is only here because it is part of the upstream vendor source.